src/core/src/FabricGeometry.cpp

```diff
@@ -181,10 +203,6 @@ void FabricGeometry::initialize() {
 }
 
 void FabricGeometry::reset() {
-    if (!UsdUtil::hasStage()) {
-        return;
-    }
-
```

Comment:
Shouldn't the `stageDestroyed` guard go here?

Reply:
It only needs to go in public member functions. `setActive` will do this check before it calls `reset`.